xline/src/server/kv_server.rs

```diff
@@ -128,23 +143,31 @@ impl Kv for KvServer {
 
         let range_request = request.into_inner();
         let propose_id = self.generate_propose_id();
-        let result = self
-            .propose(propose_id.clone(), Request::RequestRange(range_request))
-            .await;
-
-        match result {
-            Ok((res_op, sync_res)) => {
-                let mut res = Self::parse_response_op(res_op.decode());
-                let revision = sync_res.revision();
-                debug!("Get revision {:?} for RangeRequest", revision);
-                Self::update_header_revision(&mut res, revision);
-                if let Response::ResponseRange(response) = res {
-                    Ok(tonic::Response::new(response))
-                } else {
-                    panic!("Receive wrong response {:?} for RangeRequest", res);
-                }
-            }
-            Err(e) => panic!("Failed to receive response from KV storage, {e}"),
+        let is_fast_path = true;
```

Comment:
Why do we hard code is_fast_path value here?

Reply:
This is test purpose. In the future, this value should read from protobuf.